interface.go

```diff
@@ -16,7 +16,7 @@ type ProcessConfig struct {
 	CreateStdInPipe  bool
 	CreateStdOutPipe bool
 	CreateStdErrPipe bool
-	ConsoleSize      [2]int
+	ConsoleSize      [2]uint
```

Comment:
I don't think this is actually used in the platform. We might be ok to just remove it (We already set console size with the ResizeConsole method).